src/external/ocaml-rocksdb/dune

```diff
@@ -48,6 +48,7 @@
         ROCKSDB_DISABLE_BACKTRACE=1 \\
         PORTABLE=1 \\
         FORCE_SSE42=1 \\
+        DISABLE_WARNING_AS_ERROR=1 \\
```

Comment:
needed for mac build to work

Reply:
@cmr suggested we shouldn't be compiling with warnings as errors anyway since we're just building a branch